src/Sylius/Bundle/AttributeBundle/Controller/AttributeController.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\AttributeBundle\Controller;
+
+use Sylius\Bundle\AttributeBundle\AttributeType\TextAttributeType;
+use Sylius\Bundle\ResourceBundle\Controller\ResourceController;
+use Sylius\Component\Attribute\Model\AttributeInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class AttributeController extends ResourceController
+{
+    /**
+     * @return Response
+     */
+    public function renderAttributeTypesAction()
```

Comment:
How about we make this action more generic? We may want to get attribute types as JSON or XML here. Let's use FOSRestBundle to handle this. Also for the name of action and template:

``getAttributeTypesAction()``?

``attributeTypes.html.twig``?

WDYT?